torchvision/prototype/models/quantization/mobilenetv3.py

```diff
@@ -0,0 +1,100 @@
+import warnings
+from functools import partial
+from typing import Any, List, Optional, Union
+
+import torch
+from torchvision.transforms.functional import InterpolationMode
+
+from ....models.quantization.mobilenetv3 import (
+    InvertedResidualConfig,
+    QuantizableInvertedResidual,
+    QuantizableMobileNetV3,
+    _replace_relu,
+)
+from ...transforms.presets import ImageNetEval
+from .._api import Weights, WeightEntry
+from .._meta import _IMAGENET_CATEGORIES
+from ..mobilenetv3 import MobileNetV3LargeWeights, _mobilenet_v3_conf
+
+
+__all__ = [
+    "QuantizableMobileNetV3",
+    "QuantizedMobileNetV3LargeWeights",
+    "mobilenet_v3_large",
+]
+
+
+def _mobilenet_v3_model(
```

Comment:
I kept this method instead on dumping everything in the public method because on the future we might want to support also the `_small` version of the model and this method will remain unchanged.